.gitignore

```diff
@@ -4,6 +4,14 @@
 /node_modules
 /.pnp
 .pnp.js
+.pnp.*
+.yarn
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Probably need to now clean the git, since it won't track changes here, but these files are still on the branch.

Reply:
See: https://github.com/drewdrewthis/positive-blockchain-nextjs-app/tree/main/.yarn/cache